plugins/catalog-backend/src/ingestion/descriptors/DescriptorEnvelopeParser.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import yaml from 'yaml';
+import { unindent } from '../../util';
+import { makeValidator } from '../../validation';
+import { DescriptorEnvelopeParser } from './DescriptorEnvelopeParser';
+
+describe('DescriptorEnvelopeParser', () => {
+  let data: any;
+  let parser: DescriptorEnvelopeParser;
+
+  beforeEach(() => {
+    data = yaml.parse(unindent`
```

Comment:
Do you even need to unindent this? think yaml usually supports leading whitespace

Reply:
Hahaha, did I do this unnecessarily?! Maybe I should have assumed less and just tried it...

Reply:
removed...